### ISSUE TITLE
Unsupported Key

### ISSUE THREAD

**Issue by timstrasser** [lucanaef/PGPro#56]
I created a new key accordung to this guide: https://gist.github.com/ageis/14adc308087859e199912b4c79c4aaa4

So I have a 4096 bit RSA key with support Sign & Certify with two subkeys for Encrypt and Authenticate.

Then I tried to import the backup of the secret key and the backup of the secret subkeys in the PGPro Keychain, however both times a message popped up, telling me the key was unsupported.

**Comment by lucanaef** [comment]
Unfortunately sub keys don't work well in PGPro and are thus currently not supported.

**Comment by iKruton** [comment]
Any update on when this might be supported?
Having the same issue.

**Comment by lucanaef** [comment]
Not really. You can track updates of the cryptography framework PGPro uses [here](https://github.com/krzyzanowskim/ObjectivePGP).
